debian/python3-bmv2.install

```diff
@@ -0,0 +1,54 @@
+usr/lib/python*/site-packages/bm/pdfixed/conn_mgr_pd_rpc/constants.py
+usr/lib/python*/site-packages/bm/pdfixed/conn_mgr_pd_rpc/ttypes.py
+usr/lib/python*/site-packages/bm/pdfixed/mc_pd_rpc/__init__.py
+usr/lib/python*/site-packages/bm/pdfixed/mc_pd_rpc/constants.py
+usr/lib/python*/site-packages/bm/pdfixed/mc_pd_rpc/mc-remote
+usr/lib/python*/site-packages/bm/pdfixed/mc_pd_rpc/mc.py
+usr/lib/python*/site-packages/bm/pdfixed/mc_pd_rpc/ttypes.py
+usr/lib/python*/site-packages/bm/pdfixed/res_pd_rpc/
+usr/lib/python*/site-packages/bm/pdfixed/res_pd_rpc/__init__.py
+usr/lib/python*/site-packages/bm/pdfixed/res_pd_rpc/constants.py
+usr/lib/python*/site-packages/bm/pdfixed/res_pd_rpc/ttypes.py
+usr/lib/python*/site-packages/bm/pdfixed/sswitch_pd_rpc/
+usr/lib/python*/site-packages/bm/pdfixed/sswitch_pd_rpc/__init__.py
+usr/lib/python*/site-packages/bm/pdfixed/sswitch_pd_rpc/constants.py
+usr/lib/python*/site-packages/bm/pdfixed/sswitch_pd_rpc/sswitch-remote
+usr/lib/python*/site-packages/bm/pdfixed/sswitch_pd_rpc/sswitch.py
+usr/lib/python*/site-packages/bm/pdfixed/sswitch_pd_rpc/ttypes.py
+usr/lib/python*/site-packages/bm_runtime/
+usr/lib/python*/site-packages/bm_runtime/__init__.py
+usr/lib/python*/site-packages/bm_runtime/simple_pre/
+usr/lib/python*/site-packages/bm_runtime/simple_pre/SimplePre.py
+usr/lib/python*/site-packages/bm_runtime/simple_pre/__init__.py
+usr/lib/python*/site-packages/bm_runtime/simple_pre/constants.py
+usr/lib/python*/site-packages/bm_runtime/simple_pre/ttypes.py
+usr/lib/python*/site-packages/bm_runtime/simple_pre_lag/
+usr/lib/python*/site-packages/bm_runtime/simple_pre_lag/SimplePreLAG.py
+usr/lib/python*/site-packages/bm_runtime/simple_pre_lag/__init__.py
+usr/lib/python*/site-packages/bm_runtime/simple_pre_lag/constants.py
+usr/lib/python*/site-packages/bm_runtime/simple_pre_lag/ttypes.py
+usr/lib/python*/site-packages/bm_runtime/standard/
+usr/lib/python*/site-packages/bm_runtime/standard/Standard.py
+usr/lib/python*/site-packages/bm_runtime/standard/__init__.py
+usr/lib/python*/site-packages/bm_runtime/standard/constants.py
+usr/lib/python*/site-packages/bm_runtime/standard/ttypes.py
+usr/lib/python*/site-packages/bmpy_utils.py
+usr/lib/python*/site-packages/nanomsg_client.py
+usr/lib/python*/site-packages/p4dbg.py
+usr/lib/python*/site-packages/pswitch_CLI.py
+usr/lib/python*/site-packages/pswitch_runtime/
+usr/lib/python*/site-packages/pswitch_runtime/PsaSwitch.py
+usr/lib/python*/site-packages/pswitch_runtime/__init__.py
+usr/lib/python*/site-packages/pswitch_runtime/constants.py
+usr/lib/python*/site-packages/pswitch_runtime/ttypes.py
+usr/lib/python*/site-packages/runtime_CLI.py
+usr/lib/python*/site-packages/sswitch_CLI.py
+usr/lib/python*/site-packages/sswitch_runtime/
+usr/lib/python*/site-packages/sswitch_runtime/SimpleSwitch.py
+usr/lib/python*/site-packages/sswitch_runtime/__init__.py
+usr/lib/python*/site-packages/sswitch_runtime/constants.py
+usr/lib/python*/site-packages/sswitch_runtime/ttypes.py
```

Comment:
I am not sure, but believe that the default Ubuntu installs of Python3 ignore the site-packages directory, using its own dist-packages directory instead.  Good to test an install of this at least to see if it can find these packages.

Reply:
This is the location where the debian/rules "install" target puts these files. It should be the same as for a regular build with "make install DESTDIR=/usr".